platform/broadcom/saibcm-modules/systems/bde/linux/kernel/linux-kernel-bde.c

```diff
@@ -3199,12 +3199,10 @@ _init(void)
         unsigned int eb_ba = 0x0;
 
         gprintk("EB bus info: %s\n", eb_bus);
-        tok = strtok(eb_bus,",");
-        while (tok) {
+        while ((tok = strtok_r(eb_bus, ",", &eb_bus))) {
```

Comment:
>eb_bus [](http://example.com/codeflow?start=45&length=6)

You need provide a temporary pointer to store the output parameter, and it could not be `eb_bus`.
ref: [https://stackoverflow.com/a/15961298/2514803](https://stackoverflow.com/a/15961298/2514803)

Reply:
Fixed.